modules/eks/alb-controller-ingress-group/README.md

```diff
@@ -0,0 +1,145 @@
+# alb-controller-ingress-group
```

Comment:
Remove all introspection mixins and references, since we decided to not include introspection upstream. (Adding comment here, since I cant link on a renamed file)